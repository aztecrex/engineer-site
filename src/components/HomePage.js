import React from 'react';
import LatestArticle from './LatestArticle';
import Directory from './Directory';
import Harper from './Harper';

class HomePage extends React.Component {

  render() {
    return (
      <div className="page home">
        <LatestArticle />
        <hr />
        <Directory />
        <hr />
        <Harper />
      </div>
    );
  }
}


export default HomePage;
