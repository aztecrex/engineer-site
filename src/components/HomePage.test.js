import React from 'react';
import ezJson from 'enzyme-to-json';

import HomePage from './HomePage';

describe('render', () => {

  it ('renders', () => {

    // when
    let rendered = shallow(<HomePage />);

    // then
    expect(ezJson(rendered)).toMatchSnapshot();
  });


});
