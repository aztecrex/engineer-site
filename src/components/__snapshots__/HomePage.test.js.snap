exports[`render renders 1`] = `
<div
  className="page home">
  <Connect(Directory) />
  <hr />
  <Connect(LatestArticle) />
  <hr />
  <Connect(Directory) />
  <hr />
  <Connect(Harper) />
</div>
`;
